server/main.lua

```diff
@@ -0,0 +1,66 @@
+---@class InsertVehicleEntityQuery
+---@field license string
+---@field citizenId string
+---@field model string
+---@field plate string
+---@field mods string
+
+--- Creates a Vehicle DB Entity
+---@param query InsertVehicleEntityQuery Insert Vehicle Entity Request
+local function createEntity(query)
+    MySQL.insert('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state) VALUES (?,?,?,?,?,?,?)', {
+        query.license,
+        query.citizenId,
+        query.model,
+        joaat(query.model),
+        query.mods or '{}',
+        query.plate,
+        0
+    })
+end
+
+exports('CreateEntity', createEntity)
+
+---@class FetchVehicleEntityQuery
+---@field valueType 'citizenid'|'license'|'plate'
+---@field value string
+
+---@alias vehicleEntity table
+
+--- Fetches DB Vehicle Entity
+---@param query FetchVehicleEntityQuery
+---@return vehicleEntity[]
+local function fetchEntity(query)
+    if query.valueType ~= 'citizenid' and query.valueType ~= 'license' and query.valueType ~= 'plate' then return end
+    return MySQL.await('SELECT * FROM player_vehicles WHERE ? = ?', {
```

Comment:
We should transform the query results to a table so we control the format and naming before returning. For example I think we should change the name of vehicle to model before returning

Reply:
Just submitted a commit. Is that what you meant?